gobblin-runtime/src/test/java/org/apache/gobblin/runtime/api/MysqlMultiActiveLeaseArbiterTest.java

```diff
@@ -17,8 +17,10 @@
 
 package org.apache.gobblin.runtime.api;
 
+import com.google.common.base.Optional;
```

Comment:
let's use java `Optional`

(seems the original class under test mistakenly used the deprecated guava one... I presume it's not too difficult at this early stage to correct)

Reply:
The guava optional is actually used throughout the code in `FlowTriggerHandler, GobblinServiceJobScheduler, LeaseArbiter, `test class etc...I'd rather not spend as much time refactoring this right now since a lot of these pieces of code interact with one another so switching between `Java/Guava` optionals will get confusing and I'd need to change it in a lot of places.